client/components/Navbar.js

```diff
@@ -12,14 +12,15 @@ class Navbar extends React.Component {
     this.state = {
       userAvatar: 'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png',
       user: false,
+      ConditionalHomeLink: '/',
```

Comment:
no . the app first redirect to home then home redirect to dashboard. This causes a blink and a bad impression to the user.